packages/test/src/integration-tests/helpers.ts

```diff
@@ -0,0 +1,105 @@
+import { randomUUID } from 'crypto';
+import { ErrorConstructor, ExecutionContext, TestFn } from 'ava';
+import { WorkflowFailedError, WorkflowHandle, WorkflowStartOptions } from '@temporalio/client';
+import {
+  LocalTestWorkflowEnvironmentOptions,
+  TestWorkflowEnvironment,
+  workflowInterceptorModules as defaultWorkflowInterceptorModules,
+} from '@temporalio/testing';
+import {
+  DefaultLogger,
+  LogLevel,
+  Runtime,
+  WorkerOptions,
+  WorkflowBundle,
+  bundleWorkflowCode,
+} from '@temporalio/worker';
+import * as workflow from '@temporalio/workflow';
+import { ConnectionInjectorInterceptor } from '../activities/interceptors';
+import { Worker, test as anyTest, bundlerOptions } from '../helpers';
```

Comment:
Would it be difficult/disruptive to merge the content of this file into `../helpers.ts`?

Reply:
That sounds good, I'm happy for any further renaming/reorganizing tweaks to be done here; I just wanted to start getting things in shape while adding new tests for Update.